browser_use/browser/session.py

```diff
@@ -262,7 +266,7 @@ async def start(self) -> Self:
 
 				# launch/connect to the browser:
 				# setup playwright library client, Browser, and BrowserContext objects
-				await self.setup_playwright()
+				await self.setup_driver()
```

Comment:
lets leave this as `setup_playwright` for clarity, until we actually support any non-playwright-compatible drivers